src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -452,6 +453,82 @@ private string GetCategoryStringFromAttributes()
         }
 
         /// <summary>
+        ///     Dictionary Link property
+        /// </summary>
+        /// <value>
+        ///     If the node has a name and a category, convert them into a link going to the node's help page on
+        ///     Dynamo Dictionary, and return the link.
+        ///     Otherwise, return the Dynamo Dictionary home page.
+        /// </value>
+        [JsonIgnore]
+        public string DictionaryLink
+        {
+            get
+            {
+                string finalLink = Configurations.DynamoDictionary + "#/";
+                if (category == null || category == "")
+                {
+                    return finalLink; // if there is no category, return the link to home page
+                }
+
+                int i = category.LastIndexOf(Configurations.CategoryDelimiterString);
+                switch (category.Substring(i + 1))
+                {
+                    case Configurations.CategoryGroupAction:
+                        finalLink += ObtainURL(category.Substring(0, i));
+                        finalLink += "Action/";
+                        break;
+                    case Configurations.CategoryGroupCreate:
+                        finalLink += ObtainURL(category.Substring(0, i));
+                        finalLink += "Create/";
+                        break;
+                    case Configurations.CategoryGroupQuery:
+                        finalLink += ObtainURL(category.Substring(0, i));
+                        finalLink += "Query/";
+                        break;
+                    default:
+                        finalLink += ObtainURL(category);
+                        finalLink += "Action/";
+                        break;
+                }
+
+                // Obtain the node's default name from its creation name.
+                // e.g. For creation name DSCore.Math.Max@double,double - the name "Max" is obtained and appended to the final link.
+                int indexAfter = (CreationName.LastIndexOf('@') == -1) ? CreationName.Length : CreationName.LastIndexOf('@');
```

Comment:
Thanks @aparajit-pratap , it appears that some nodes have empty strings as their `Name`. Though, the actual name is still obtainable from `CreationName`, something which I don't quite understand as `CreationName` seems to return the node's `Name` property from their `get` method (line 77 of the code in this file).
`Nickname` changes accordingly with the UI when the user renames the node. Hence, I think that `CreationName` gives a better accuracy among the three.